assets/src/edit-story/components/canvas/canvasUploadDropTarget.js

```diff
@@ -49,17 +50,24 @@ const PageAreaCover = styled(PageArea)`
 
 function CanvasUploadDropTarget({ children }) {
   const { uploadFile } = useUploader();
+  const { showSnackbar } = useSnackbar();
   const insertElement = useInsertElement();
   const onDropHandler = useCallback(
     (files) => {
       files.forEach((file) => {
-        uploadFile(file).then((res) => {
-          const resource = getResourceFromUploadAPI(res);
-          insertElement(resource.type, { resource });
-        });
+        uploadFile(file)
+          .then((res) => {
+            const resource = getResourceFromUploadAPI(res);
+            insertElement(resource.type, { resource });
+          })
+          .catch((e) => {
+            showSnackbar({
+              message: e.message,
```

Comment:
Not sure if this is ok for "alpha". Basically, the fork here is this:

```
message = e.isUserReadableError ? e.message : __('Some random error happened. Not sure what you can do')
```

That sort of thing. Showing a user "DOMException: Abort: illegal invocation" is not going to be helpful :)

Reply:
Yes, so for now add `isUserError` flag for `validError` and `sizeError` and if not added `__('Sorry, file has been failed to upload....)`